src/server.rs

```diff
@@ -285,7 +297,12 @@ impl RollupBoostServer {
 
         // TODO: Use _is_block_building_call to log the correct message during the async call to builder
         let (should_send_to_builder, _is_block_building_call) =
-            if let Some(attr) = payload_attributes.as_ref() {
+            if *self.num_invalid_builder_payloads.lock().await
+                >= self.config.max_allowed_invalid_builder_payloads
+            {
+                // Circuit breaker is triggered, do not send to builder
+                (false, false)
```

Comment:
@ferranbt for simplicity I set `_is_block_building_call` to false here. what uses did you have in mind for this variable? Is it fine to set to false when the circuit breaker is hit?